test/cmd/basicresources.sh

```diff
@@ -338,7 +338,6 @@ os::cmd::expect_failure_and_text "oc new-project --token="$( oc sa get-token bui
 
 # test oc projects
 os::cmd::expect_failure_and_text 'oc projects test_arg' 'no arguments'
-os::cmd::expect_success_and_text 'oc projects' 'You have access'
```

Comment:
This test has no value -- there are tests to ensure that `oc projects` outputs the correct things for single and multiple projects below. Furthermore, this test *only* works if `login.sh` is run first, as the second project it sees (and therefore the project that causes the `You have access` text) is one that's created in `login.sh` but not cleaned up.